journal/updated/03-chapter-2.adoc

```diff
@@ -198,49 +203,48 @@ It seems very hard for me to tell you that I am yet bound in this place,
 although when I wrote to you I thought that I might have got away.
 It is not omission of duty that detains me,
 but surely it is for a trial of my patience and of yours, for a cause we know not.
-I fear we have been over anxious;
+I fear we have been over-anxious,
 yet sometimes I have hopes that you are more resigned
 to the will of the Most High than I am.
 If I be rightly sensible, I believe that I am where I ought to be;
-and although I would have hid the concern, yet I could not;
+and although I would have hid the concern, yet I could not,
 for several Friends whose spirits are united with mine,
-and who had a sense of my travail of soul came to see me--and my dear friends
-with whom I make this long abode told me they did believe I was not clear,
-and my unwillingness to stay made my affliction heavier; which indeed was true.
+and who had a sense of my travail of soul, came to see me. And my dear friends
```

Comment:
I know its a bit weird, but I think we need to leave the double-hyphen here, because its all part of one thought, one sentence in her mind, explaining why she thinks she's in her right place.  So lets put it back to: .... came to see me--and my dear friends